packages/@webex/plugin-meetings/src/meeting-info/index.ts

```diff
@@ -71,20 +71,32 @@ export default class MeetingInfo {
    * @memberof MeetingInfo
    */
   private requestFetchInfo(options: any) {
-    const {meetingId} = options;
-    if (meetingId) {
+    const {meetingId, hasPrejoinStarted} = options;
+    if (meetingId && hasPrejoinStarted) {
```

Comment:
are we sure that we want to send both 'internal.client.meetinginfo.request' and 'client.meetinginfo.request' only if `hasPrejoinStarted` is true? If that's the case, then it looks like we are sending these 2 events in all the same places so why do we need the `internal.client.meetinginfo.request` at all? We could just update getMeetingInfoReqResp() to use 'client.meetinginfo.request' and 'client.meetinginfo.response' instead of the internal events.

Reply:
changing to sending internal.client.meetinginfo.request as before: if there is meeting.id

Reply:
internal.client.meetinginfo.request tracks latencies of meeting info req/res whether in prejoin or in join phase. So it should always be sent as long as there is a meeting.id

client.meetinginfo.request is only for prejoin, so should only be sent if there is a correlation id and a meeting.id